jib-core/src/main/java/com/google/cloud/tools/jib/http/Connection.java

```diff
@@ -46,14 +48,37 @@
 public class Connection implements Closeable {
 
   /**
-   * Do not use {@link NetHttpTransport}. It does not process response errors properly. A new {@link
-   * ApacheHttpTransport} needs to be created for each connection because otherwise HTTP connection
-   * persistence causes the connection to throw {@link NoHttpResponseException}.
+   * Returns a factory for {@link Connection}.
    *
-   * @see <a
-   *     href="https://github.com/google/google-http-java-client/issues/39">https://github.com/google/google-http-java-client/issues/39</a>
+   * @return {@link Connection} factory
```

Comment:
nit: maybe include a description of what the factory is (like "a function that generates a Connection to a URL")